app/models/grda_warehouse/project_group.rb

```diff
@@ -38,6 +38,12 @@ class ProjectGroup < GrdaWarehouseBase
       viewable_by(user)
     end
 
+    scope :text_search, -> (text) do
+      return none unless text.present?
+  
+      where(arel_table[:name].lower.matches("%#{text.downcase}%"))
```

Comment:
This should also search the `ProjectName` of any associated projects.  Something like:

```
joins(:projects).where(arel_table[:name].lower.matches("%#{text.downcase}%").or(
p_t[:ProjectName].lower.matches("%#{text.downcase}%"))
```

That's probably not exactly right, but should be close.